quokka_editor_front/src/components/atoms/searchBar/SearchBar.tsx

```diff
@@ -1,10 +1,35 @@
+import { getPageOfDocuments } from "../../../api";
+import { useEffect, useState } from "react";
+import { useAppDispatch } from "../../../Redux/hooks";
+import { getDocuments } from "../../../Redux/documentsSlice";
+import { DEFAULT_PAGE_PARAMS } from "../../../consts";
+
 const SearchBar: React.FC = () => {
```

Comment:
Did you use debouncing?

Reply:
Searching didn't work properly with debouncing/throttling so for now I don't use it